src/classes/TestAmountToText.cls

```diff
@@ -0,0 +1,72 @@
+/**
+ * Created by ociszek001 on 21.05.2019.
+ */
+@isTest
+private class TestAmountToText {
+
+    @TestSetup
+    static void loadInvoices() {
+        User u = TestDataClass.insertUser();
+        List<Invoice__c> inv = new List<Invoice__c>();
+        System.runAs(u) {
+            Test.startTest();
+            VAT__c vat = TestDataClass.insertVat();
+            inv = TestDataClass.loadInvoices();
+            List<Service__c> ser = TestDataClass.insertServices(new List<VAT__c>{
+                    vat
+            }, inv);
+            Test.stopTest();
+        }
+        System.assertEquals(false, inv.isEmpty());
+    }
+    @isTest
+    static void TestAmountToText() {
+        User u = [SELECT Id from User WHERE userName = :TestDataClass.userName];
+        List<Invoice__c> inv = new List <Invoice__c>();
+        System.runAs(u) {
+            Test.startTest();
+            List<Service__c> ser = [SELECT Id, Invoice__c, Net_Price__c from Service__c];
```

Comment:
Uporządkujmy ten kod bo jest napisany bardzo niechlujnie, albo deklaracja List<Invoice__c> inv do środka albo  List<Service__c> ser na zewnątrz. Uporządkuj proszę we wszystkich miejscach.